ethereum/forge-test/GasBenchmark.t.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "forge-std/Test.sol";
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
+import "./utils/WormholeTestUtils.t.sol";
```

Comment:
Is this `*.t.*` convention something taken from foundry? Curious what the significance of this is?

If it's optional, I wonder if it's necessary, given that these contracts are already under `forge-test`.

Reply:
Yeah it's a foundry convention. `*.t.sol` means a test contract. `*.s.sol` means a script contract. Although they are usually placed in test and script directories, but they could be anywhere.

I don't know what is the rational here but I like to follow the practice. I can think it's good when we are having test contracts in the source as unit tests. Like rust, I can think of test contracts in the test directory as integration tests.